sc/shared/src/test/scala/sigma/LanguageSpecificationV6.scala

```diff
@@ -1522,6 +1527,148 @@ class LanguageSpecificationV6 extends LanguageSpecificationBase { suite =>
     )
   }
 
+  property("Context.getVar and getVarFromInput") {
```

Comment:
Better use separate `property` for each method.

Reply:
splitted